### DISCUSSION TITLE
Is it possible to search in All Documents by default ?

### DISCUSSION THREAD

**Discussion by nejib1** [open-webui/open-webui#728] [Q&A]
Hello,

I want to use Ollama-webui to chat with **Mistral + All Documents**.

Is it possible to modify the code (by myself not in git) to automatically have Ollama-webui **always search in All Documents without needing to type "#All Documents"** in every message?

Thank you in advance for your help.

**Comment by tjbck** [comment]
Related https://github.com/open-webui/open-webui/issues/529
